app/services/service_discovery/import_cluster_definitions_service.rb

```diff
@@ -9,7 +9,7 @@ def self.build_service(account, attributes = {})
     end
 
     def self.create_service(account, cluster_namespace:, cluster_service_name:, user: nil)
-      CreateServiceWorker.perform_async(account.id, cluster_namespace, cluster_service_name, user&.id)
+      CreateServiceWorker.perform_async(account.id, cluster_namespace.as_json, cluster_service_name.as_json, user&.id)
```

Comment:
I am thinking - would it make more sense to make it `.to_s` ? 

The result would be the same, I think, but I think `.as_json` kind of suggests that it can be a more complex data structure, while in this case `cluster_namespace` and `cluster_service_name` are strings.

WDYT @akostadinov ?

Reply:
I'm fine either way.

Reply:
I am not sure why it was added in the first place, these values will always be strings, not symbols. 

They are coming from here:
- namespace: https://github.com/3scale/porta/blob/master/app/controllers/provider/admin/service_discovery/cluster_projects_controller.rb#L5-L6 (`/p/admin/service_discovery/projects.json`)
- services: https://github.com/3scale/porta/blob/master/app/controllers/provider/admin/service_discovery/cluster_services_controller.rb#L5-L6 (`/p/admin/service_discovery/namespaces/{whatever}/services.json`)

But I think it does not harm to add `.to_s` just in case.

Reply:
@akostadinov you're right, calling `as_json` ensures the same behavior as in Sidekiq 5, but removing the deprecation warning. That's why I'm calling it, but @mayorova if you're sure they are always strings, then yes, `to_s` is good for this case.